.vscode/extensions.json

```diff
@@ -1,25 +1,17 @@
 {
     "recommendations": [
-        // "Required"
-        "ms-python.python",
         "davidanson.vscode-markdownlint",
         "stylelint.vscode-stylelint",
         "redhat.vscode-yaml",
-        "batisteo.vscode-django",
         "octref.vetur",
         "dbaeumer.vscode-eslint",
-        "EditorConfig.EditorConfig",
+        "editorconfig.editorconfig",
         "mikestead.dotenv",
-
-        // Recommended
         "esbenp.prettier-vscode",
         "lokalise.i18n-ally",
-        "eamodio.gitlens",
```

Comment:
This is a personal productivity tool for some, but not directly needed for this project. Personally an extension I don't like or use.